app/views/users/show.html.erb

```diff
@@ -1 +1,59 @@
-<h1>Here is the info for a particular post</h1>
+<%= stylesheet_link_tag 'application', 'usershow', 'data-turbo-track': 'reload' %>
+
+<body>
+  <main class="flex">
+    <header></header>
+
+    <div class="profile">
+      <span class="profile-photo">
+        <img src="https://user-images.githubusercontent.com/37116322/222183427-3450696b-429a-4e9a-9e19-48bbe4b48edf.png" alt="profile" class="user-pic">
+      </span>
+      <span class="user">
+        <h2 class="name"><%= @user.name %></h2>
+        <span class="second-half">
+          <h5>#Number of posts:</h5>
+          <h6 class="numberofposts"><%= @user.posts_counter %></h6>
+        </span>
+      </span>
+    </div>
+
+    <div class="bio">
+      <h3 class="bio-title">Bio</h3>
+      <p class="bio-content">
+        <%= @user.bio %>
+      </p>
+    </div>
+
+    <% @user.posts.each do |post| %>
```

Comment:
- [ ] Here you are supposed to show the user's recent posts i.e 3 posts, currently, it shows all post kindly fix this